handle_args: char:2; exception: E2().
----------------
Coverage Report:

__main__: test/exc-TA_{:d}.py:
   2   from exc import *
   3
   4   def top(arg):
   5     try:
   6 %     try_(arg)
   7     except E1 as e1:

__main__: test/exc-TA_{:d}.py: 10 lines; 3 trivial; 7 traceable; 6 covered; 0 ignored; 0 ignored but covered; 1 not covered.
